packages/test/test-gc-sweep-tests/src/test/mockSummarizerClient.ts

```diff
@@ -0,0 +1,206 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DataObject } from "@fluidframework/aqueduct";
+import { assert, TelemetryNullLogger } from "@fluidframework/common-utils";
+import { IRuntimeFactory, LoaderHeader } from "@fluidframework/container-definitions";
+import {
+    ContainerRuntime,
+    gcBlobPrefix,
+    gcTreeKey,
+    ISummaryNackMessage,
+    SummaryCollection,
+    neverCancelledSummaryToken,
+    ISummaryCancellationToken,
+    SummarizerStopReason,
+} from "@fluidframework/container-runtime";
+import { IFluidDataStoreRuntime } from "@fluidframework/datastore-definitions";
+import { DriverHeader } from "@fluidframework/driver-definitions";
+import { concatGarbageCollectionStates } from "@fluidframework/garbage-collector";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import { ITestObjectProvider } from "@fluidframework/test-utils";
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
+import { IGarbageCollectionState } from "@fluidframework/runtime-definitions";
+import { ILoaderProps } from "@fluidframework/container-loader";
+
+// data store that exposes container runtime for testing.
+export class TestDataObject extends DataObject {
+    public get _root() {
+        return this.root;
+    }
+
+    public get dataStoreRuntime(): IFluidDataStoreRuntime {
+        return this.runtime;
+    }
+
+    public get containerRuntime(): ContainerRuntime {
+        return this.context.containerRuntime as ContainerRuntime;
+    }
+
+    public get _context() {
+        return this.context;
+    }
+}
+
```

Comment:
Doesn't look like anything below here in the file is used, save for a future PR?